### PR TITLE
StoryBook Package added and default Button for reference

### PR DESCRIPTION
**Pull request by Sameer-Begh**

### CODE REVIEW

File: src/components/ui/button/styles.scss [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  cursor: pointer;
+  display: inline-block;
+  line-height: 1;
+}
+.storybook-button--primary {
+  color: white;
+  background-color: #1ea7fd;
+}
+.storybook-button--secondary {
+  color: #333;
```
Comment (shelly19gkmit) [inline src/components/ui/button/styles.scss:15]: use constants for colors

File: src/components/ui/button/Button.stories.ts [hunk 2]
```diff
@@ -0,0 +1,52 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import { fn } from '@storybook/test';
+import Button from '../../ui/button/index';
```
Comment (shelly19gkmit) [inline src/components/ui/button/Button.stories.ts:3]: Can we make Alias as well for folders

File: .eslintrc.json [hunk 3]
```diff
@@ -1,3 +1,6 @@
 {
-  "extends": "next/core-web-vitals"
+  "extends": [
```
Comment (shelly19gkmit) [inline .eslintrc.json:2]: Are we directly raising PRs against main not staging?

File: src/components/ui/button/styles.scss (hunk 1 above)
Comment (Sameer-Begh) [reply src/components/ui/button/styles.scss:15]: Not able to use color from constant file in css file

File: .eslintrc.json (hunk 3 above)
Comment (Sameer-Begh) [reply .eslintrc.json:2]: done

File: src/components/ui/button/Button.stories.ts (hunk 2 above)
Comment (Sameer-Begh) [reply src/components/ui/button/Button.stories.ts:3]: We can do once folder structure ready

File: src/components/ui/button/styles.scss (hunk 1 above)
Comment (shelly19gkmit) [reply src/components/ui/button/styles.scss:15]: check if we can use variables just like in Indigg repo

File: src/components/ui/button/Button.stories.ts (hunk 2 above)
Comment (shelly19gkmit) [reply src/components/ui/button/Button.stories.ts:3]: okay

File: src/components/ui/button/Button.stories.ts [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+// More on how to set up stories at: https://storybook.js.org/docs/writing-stories#default-export
+const meta = {
+  title: 'Componets/Button',
```
[the remark this answers is not in the document]
Comment (shelly19gkmit) [reply src/components/ui/button/Button.stories.ts:7]: Why marked as resolved? Spelling is not correct

**Review by mandeepatindigg — APPROVED** [review]

**Pull request merged**